From 6ac591c244e2da51d7ffc7929de80f842d286309 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?G=C3=B6k=C3=A7en=20Eraslan?= <gokcen@pardus.org.tr>
Date: Thu, 3 Feb 2011 08:45:33 +0000
Subject: [PATCH] Enable autoconnect by default.

Autoconnect is enabled by default in NM Settings Spec and in NM Gnome
UI.
---
 libs/internals/connection.cpp |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libs/internals/connection.cpp b/libs/internals/connection.cpp
index 9048533..e1925d3 100644
--- a/libs/internals/connection.cpp
+++ b/libs/internals/connection.cpp
@@ -87,13 +87,13 @@ Connection::Type Connection::typeFromString(const QString & typeString)
 Connection::Type typeFromString(const QString & type);
 
 Connection::Connection(const QString & name, const Connection::Type type, const Connection::Scope scope)
-    : m_name(name), m_uuid(QUuid::createUuid()), m_type(type), m_scope(scope), m_autoConnect(false)
+    : m_name(name), m_uuid(QUuid::createUuid()), m_type(type), m_scope(scope), m_autoConnect(true)
 {
     init();
 }
 
 Connection::Connection(const QUuid & uuid, const Connection::Type type)
-    : m_uuid(uuid), m_type(type), m_autoConnect(false)
+    : m_uuid(uuid), m_type(type), m_autoConnect(true)
 {
     init();
 }
-- 
1.7.3.4

